### PR TITLE
Arabic translations for image editor

### PR DESCRIPTION
**Pull request by pal-tgiannakopoulos**

### CODE REVIEW

File: ananas/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
-    <string name="iamutkarshtiwari_github_io_ananas_text">_Text</string>
-    <string name="iamutkarshtiwari_github_io_ananas_brush">_Brush</string>
-    <string name="iamutkarshtiwari_github_io_ananas_eraser">_Eraser</string>
-    <string name="iamutkarshtiwari_github_io_ananas_edit">_Edit</string>
-    <string name="iamutkarshtiwari_github_io_ananas_brightness">_Brightness</string>
-    <string name="iamutkarshtiwari_github_io_ananas_saturation">_Saturation</string>
-    <string name="iamutkarshtiwari_github_io_ananas_size">_Size</string>
-    <string name="iamutkarshtiwari_github_io_ananas_opacity">_Opacity</string>
+    <string name="iamutkarshtiwari_github_io_ananas_stickers">ملصقات</string>
+    <string name="iamutkarshtiwari_github_io_ananas_filter">فلاتر</string>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml:9]: What if we called it "مؤثرات" instead?

File: ananas/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -1,65 +1,65 @@
...
+    <string name="iamutkarshtiwari_github_io_ananas_crop">قص</string>
+    <string name="iamutkarshtiwari_github_io_ananas_rotate">تدوير</string>
+    <string name="iamutkarshtiwari_github_io_ananas_text">نص</string>
+    <string name="iamutkarshtiwari_github_io_ananas_brush">فرشاة</string>
+    <string name="iamutkarshtiwari_github_io_ananas_eraser">ممحاة</string>
+    <string name="iamutkarshtiwari_github_io_ananas_edit">تحرير</string>
+    <string name="iamutkarshtiwari_github_io_ananas_brightness">الإضاءة</string>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml]: I think it's better to call this "السطوع" which is what I've seen in the previous editor as well.

File: ananas/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -1,65 +1,65 @@
...
+    <string name="iamutkarshtiwari_github_io_ananas_saturation">تشبع اللون</string>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml]: Previous editor called it "التشبع" should we mention colour as well?

File: ananas/src/main/res/values-ar/strings.xml [hunk 4]
```diff
@@ -1,65 +1,65 @@
...
+    <string name="iamutkarshtiwari_github_io_ananas_loading">جاري التحميل...</string>
 
-    <string name="iamutkarshtiwari_github_io_ananas_free_size">_FREE</string>
-    <string name="iamutkarshtiwari_github_io_ananas_fit_image">_ORIGINAL</string>
-    <string name="iamutkarshtiwari_github_io_ananas_square">_SQUARE</string>
-    <string name="iamutkarshtiwari_github_io_ananas_ratio3_4">_3:4</string>
-    <string name="iamutkarshtiwari_github_io_ananas_ratio4_3">_4:3</string>
-    <string name="iamutkarshtiwari_github_io_ananas_ratio9_16">_9:16</string>
-    <string name="iamutkarshtiwari_github_io_ananas_ratio16_9">_16:9</string>
+    <string name="iamutkarshtiwari_github_io_ananas_free_size">حجم غير محدود</string>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml]: "بلا قيود" or "غير مقيد" what do you think?

File: ananas/src/main/res/values-ar/strings.xml [hunk 5]
```diff
@@ -1,65 +1,65 @@
...
-        <item>_Memory</item>
-        <item>_Yogurt</item>
-        <item>_Fleeting</item>
-        <item>_Shine</item>
+        <item>أصلي</item>
+        <item>ناعم</item>
+        <item>أسود</item>
+        <item>كلاسيكي</item>
+        <item>خلاب</item>
+        <item>ريترو</item>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml]: What do you think of "تقليدي"?

File: ananas/src/main/res/values-ar/strings.xml [hunk 6]
```diff
@@ -1,65 +1,65 @@
...
+        <item>أنيق</item>
+        <item>فيلم</item>
+        <item>ذكرى</item>
+        <item>لبن</item>
+        <item>عابر</item>
+        <item>مضيء</item>
```
Comment (pal-ifadlalla) [inline ananas/src/main/res/values-ar/strings.xml]: What about "متألق" or "مشرق"?

**Review by Kate-sj — APPROVED** [review]

**Review by pal-ifadlalla — APPROVED** [review]
Looks good to me!

**Pull request merged**